index.js

```diff
@@ -69,11 +69,12 @@ function parseTorrentRemote (torrentId, cb) {
     })
   } else if (typeof get === 'function' && /^https?:/.test(torrentId)) {
     // http, or https url to torrent file
-    get.concat({
+    options = Object.assign({
```

Comment:
Please call this `opts` instead of `options`

Reply:
ok, I have corrected it.

Reply:
Thanks! Can you also address my other comment?